package.json

```diff
@@ -18,12 +18,14 @@
     "pretest": "npm run lint",
     "prewatch:browser": "npm run clean:browser",
     "prewatch:mobile": "npm run clean:mobile",
-    "lint": "standard 'src/**/*.js' 'src/**/*.jsx' 'mobile/src/**/*.js' 'mobile/src/**/*.jsx'",
+    "lint": "npm-run-all --parallel 'lint:*'",
+    "lint:styles": "stylint src/styles --config ./.stylintrc",
+    "lint:js": "standard 'src/**/*.js' 'src/**/*.jsx' 'test/**/*.js' 'mobile/src/**/*.js' 'mobile/src/**/*.jsx' 'mobile/test/**/*.js'",
     "test": "node --harmony_proxies node_modules/.bin/jest",
     "watch": "webpack --config webpack.config.js --watch --display-modules --display-chunks",
```

Comment:
I suggest to let as this and describe the modus operandi in `README.md` like

> if you want to watch files' changes and rebuild, use `yarn watch:browser` or `yarn watch:mobile`. (you don't have to run `yarn watch`. Because this project contains two different workflows, you muse choose one of them.)